### PR TITLE
Lesson 7 (KnockOut)

### PR DESCRIPTION
**Pull request by frayeralex**
add new component timerComponent

### CODE REVIEW

File: app/code/Boyko/Timer/view/frontend/web/js/storage.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+define(['ko'], function(ko) {
+    'use strict';
+
+    var _store = {};
+
+    function has(key) {
```
Comment (chabanenk0) [inline app/code/Boyko/Timer/view/frontend/web/js/storage.js]: I would advice to rename this method to hasObservable(key)

File: app/code/Boyko/Timer/registration.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (chabanenk0) [inline app/code/Boyko/Timer/registration.php]: remove this phpstorm autocomments

File: app/code/Boyko/Timer/view/frontend/web/js/storage.js (hunk 1 above)
Comment (frayeralex) [reply app/code/Boyko/Timer/view/frontend/web/js/storage.js]: fixed

File: app/code/Boyko/Timer/registration.php (hunk 2 above)
Comment (frayeralex) [reply app/code/Boyko/Timer/registration.php]: fixed

**Pull request merged**